package.json

```diff
@@ -24,5 +26,22 @@
   "bugs": {
     "url": "https://github.com/testiumjs/testium-core/issues"
   },
-  "homepage": "https://github.com/testiumjs/testium-core#readme"
+  "homepage": "https://github.com/testiumjs/testium-core#readme",
+  "devDependencies": {
+    "npub": "^2.2.0",
+    "tap": "^1.4.0"
```

Comment:
First and foremost it's "tap tends to be less verbose when using Javascript". Also (quoting myself):

> * tests that do nothing fail
> * files without tests are marked as pending
> * tap reports assertions individually
> * doesn't have to do a fatal failure on every failed assertion
> * also: not a single magical global :heart:

I'm not feeling strongly about it, I'd be fine with using mocha as well. I'm just a bit excited about using tap to implement things like "ok - element visible after 143ms". But for that we only need a tap adaptor, not write the core tests with tap.